pkg/cases/handler.go

```diff
@@ -8,14 +8,146 @@ import (
 type CaseHandler struct {
 }
 
-// @Summary Get case by case id
+// @Summary Get the lastest case
 // @Description
+// @Tags Case
 // @Produce json
-// @Param  id path uint true "case ID"
+// @Param patientID path uint true "patient ID"
 // @Success 200 {object} api.ReturnedData{data=Case}
-// @Router /case/{id} [GET]
-func (h *CaseHandler) GetCaseByCaseID(c echo.Context) error {
-	// what is in here does not matter for now, just filling the swaggo comments and name the function
-	c.Logger().Debug("hello world")
-	return c.JSON(200, api.Return("pong", nil))
+// @Router /patient/{patientID}/case [GET]
+func (h *CaseHandler) GetLastCase(c echo.Context) error {
+	// ...
+	c.Logger().Debug("GetLastCase")
+	return c.JSON(200, api.Return("success", nil))
+}
+
+// @Summary Get cases by patient ID
+// @Description
+// @Tags Case
+// @Produce json
+// @Param patientID path uint true "patient ID"
+// @Param department query string false "department name" nil
+// @Success 200 {array} api.ReturnedData{data=[]uint}
```

Comment:
```suggestion
// @Success 200 {array} api.ReturnedData{data=[]Cases}
```

Using pagination if you fear there may be too many cases